packages/account/src/Sections/Profile/PhoneVerification/phone-number-verified-modal.tsx

```diff
@@ -6,41 +6,47 @@ import { routes } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 import { LabelPairedBadgeCheckLgRegularIcon } from '@deriv/quill-icons';
 
-const PhoneNumberVerifiedModal = observer(() => {
-    const [should_show_phone_number_verified_modal, setShouldShowPhoneNumberVerifiedModal] = React.useState(false);
-    const history = useHistory();
-    const handleDoneButton = () => {
-        setShouldShowPhoneNumberVerifiedModal(false);
-        history.push(routes.personal_details);
-    };
-    const { ui } = useStore();
-    const { is_mobile } = ui;
+type TPhoneNumberVerifiedModal = {
+    should_show_phone_number_verified_modal: boolean;
+    setShouldShowPhoneNumberVerifiedModal: (value: boolean) => void;
+};
 
-    return (
-        <Modal
-            isMobile={is_mobile}
-            showHandleBar
-            isOpened={should_show_phone_number_verified_modal}
-            primaryButtonCallback={handleDoneButton}
-            primaryButtonLabel={<Localize i18n_default_text='Done' />}
-            disableCloseOnOverlay
-        >
-            <Modal.Header
-                className='phone-verification__verified-modal--header'
-                image={<LabelPairedBadgeCheckLgRegularIcon fill='#007A22' height={96} width={96} />}
-            />
-            <Modal.Body>
-                <div className='phone-verification__verified-modal--contents'>
-                    <Text bold>
-                        <Localize i18n_default_text='Verification successful' />
-                    </Text>
-                    <Text>
-                        <Localize i18n_default_text="That's it! Your number is verified." />
-                    </Text>
-                </div>
-            </Modal.Body>
-        </Modal>
-    );
-});
+const PhoneNumberVerifiedModal = observer(
+    ({ should_show_phone_number_verified_modal, setShouldShowPhoneNumberVerifiedModal }: TPhoneNumberVerifiedModal) => {
+        const history = useHistory();
+        const handleDoneButton = () => {
+            setShouldShowPhoneNumberVerifiedModal(false);
+            history.push(routes.personal_details);
+        };
+        const { ui } = useStore();
+        const { is_mobile } = ui;
+
+        return (
+            <Modal
+                isMobile={is_mobile}
+                showHandleBar
+                isOpened={should_show_phone_number_verified_modal}
+                primaryButtonCallback={handleDoneButton}
+                primaryButtonLabel={<Localize i18n_default_text='Done' />}
+                disableCloseOnOverlay
+            >
+                <Modal.Header
+                    className='phone-verification__verified-modal--header'
+                    image={<LabelPairedBadgeCheckLgRegularIcon fill='#007A22' height={96} width={96} />}
```

Comment:
will size='2xl' work here?

Reply:
I've checked there's no `size` props for this Icon 🙏